### PR TITLE
Switch name to name_ja on ja locale

### LINKED ISSUE #121: Swap to `name_ja` for Japan locale
**Is your feature request related to a problem? Please describe.**
Forgot to implement this, but we should use `name_ja` for the speaker name when the locale is `ja`.

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #121 

Use the Japanese name when locale is 'ja'.

**Testing plan:**

1. Load the main page and scroll down to view the speakers
2. Toggle the locale to Japanese
3. Check that the speakers' names are in Japanese
4. Toggle locale back to English
5. Check that the speakers' names are in English

### CODE REVIEW

File: web/src/components/SpeakerCard.vue [hunk 1]
```diff
@@ -72,6 +72,9 @@ export default {
     },
     name() {
       try {
+        if (this.$i18n.locale === 'ja') {
+          return this.speaker.get('name_ja') || '';
```
Comment (tuttiq) [inline web/src/components/SpeakerCard.vue:76]: If there's no Japanese name, shouldn't it fallback to English instead of empty string?

Comment (ann-kilzer) [reply web/src/components/SpeakerCard.vue:76]: True... let me do a little more investigation of what airtable returns on a blank record, if that kind of error can happen.

Comment (ann-kilzer) [reply web/src/components/SpeakerCard.vue:76]: From the docs: "Any "empty" fields (e.g. "", [], or false) in the record will not be returned."

File: web/src/components/SpeakerCard.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
+<template>
```
Comment (tuttiq) [inline web/src/components/SpeakerCard.vue:1]: Wait, I just realize this is a whole new file. The change on `src/components/speaker/SpeakerCard.vue` looks good to me, but why do we have this duplicate `SpeakerCard.vue` here?

Comment (ann-kilzer) [reply web/src/components/SpeakerCard.vue:1]: I made a merge mistake... lemme check

Comment (ann-kilzer) [reply web/src/components/SpeakerCard.vue]: Okay fixed. Good catch!

**Review by tuttiq — APPROVED** [review]
🎉

**Pull request merged**